integration_tests/pages/page.ts

```diff
@@ -165,6 +165,28 @@ export default abstract class Page {
     })
   }
 
+  shouldHaveErrors(errors: Array<{ field: string; message: string }>): void {
```

Comment:
I like this 👍

Reply:
The error summary bit is inspired by an earlier draft of John's PR